### DISCUSSION TITLE
Welcome to SleekDB Discussions!

### DISCUSSION THREAD

**Discussion by rakibtg** [SleekDB/SleekDB#248] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by valenzine** [comment]
I'm so, so very glad that this project is once again being updated!

I love it and I've been using it on many personal projects and I was worried I had to go look for an alternative. Thank you!

**Reply by rakibtg** [reply]
I really appreciate your feedback, thank you for being an active member :)
